System/GetRUsage.hsc

```diff
@@ -0,0 +1,39 @@
+{-# LANGUAGE CPP, NondecreasingIndentation, CApiFFI #-}
```

Comment:
Why is `NondecreasingIndentation` needed?

Reply:
That's mostly for the (unindented) `do` block starting on the line 28, which I mostly copied over from a module in `base`. yikes, but it's either this pragma or reformatting the sources (I'm fine with both, let me know if you'd rather like adding the indentation).